InfluxPublisher/tests/test_publisher_e2e.sh

```diff
@@ -0,0 +1,125 @@
+
```

Comment:
Maybe it would be good practice to add this here?
```suggestion
set -euo pipefail
```